policyengine_us/parameters/gov/states/la/tax/income/exempt_income/retirement/cap.yaml

```diff
@@ -9,6 +9,7 @@ brackets:
       2001-01-01: 65
     amount:
       2001-01-01: 6_000
+      2025-01-01: 12_000
```

Comment:
![LA retirement and disability exemption](https://github.com/user-attachments/assets/688234b5-1272-4eaf-a6b0-7c52349d8e67)

Wouldn't this affect the disability cap too?

Reply:
Yep, adjusted